news/_posts/2019-05-29-severity-classification.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "Introducing a new bug severity classification"
+subtitle: "Discover how issues related to PrestaShop software are qualified and prioritized"
+date:   2019-05-29 11:30:00
+authors: [ MarionFrancois]
+icon: icon-code
+tags:
+ - qa
+---
+
+According to the [Information Technology Infrastructure Library (ITIL)](https://fr.wikipedia.org/wiki/Information_Technology_Infrastructure_Library), a bug is an "event that is not part of the standard operation of a service and causes an unplanned interruption or decrease the quality of service". And it is everybody’s job to chase them all!
+
+At the PrestaShop Quality Assurance team (aka QA), we browse issues on GitHub all day long. As we go about doing that, we often come across messages from people who are angry about a bug taking a long time to be fixed, and saying "this is a major issue, why is it still not fixed?”, or “This bug is marked as minor but it should be major”. Everyone agrees that not all bugs can be fixed at once, and that the more critical bugs should be fixed first. The problem is that since this appreciation is subjective, any opinion will almost always be challenged by the people who are personally affected by that specific issue, and who want to get it resolved first.
```

Comment:
Maybe "angry" is a bit strong, what about "surprised", "puzzled", "who do not understand"

Reply:
I think the only fair alternative which is still true to the reality would be "annoyed"

Reply:
You will agree that there are many angry messages by angry people about not fixed bugs. I think this is why this word has been chosen, to reflect the feeling.

Reply:
Yes, annoyed seems good

Reply:
```suggestion
At the PrestaShop Quality Assurance team (aka QA), we browse issues on GitHub all day long. As we go about doing that, we often come across messages from people who are annoyed about a bug taking a long time to be fixed, and saying "this is a major issue, why is it still not fixed?”, or “This bug is marked as minor but it should be major”. Everyone agrees that not all bugs can be fixed at once, and that the more critical bugs should be fixed first. The problem is that since this appreciation is subjective, any opinion will almost always be challenged by the people who are personally affected by that specific issue, and who want to get it resolved first.
```